Uml/UseCases/UseCase.cs

```diff
@@ -22,32 +22,42 @@
 namespace Uml.UseCases
 {
     using Uml.Assembler;
+    using Uml.Attributes;
     using Uml.Classification;
     using Uml.SimpleClassifiers;
 
     /// <summary>
     /// A <see cref="UseCase"/> specifies a set of actions performed by its subjects, which yields an observable result that is of value for one or more Actors or other stakeholders of each subject.
     /// </summary>
+    [Class(IsAbstract = false, IsActive = false, Specializations = "")]

```

Comment:
It would seem that the `Specialization` property is almost always an empty string. Possibly better to make that default value of the attribute to stop repeating it. Just my 2 cents.